python/curriculum.py

```diff
@@ -394,6 +394,7 @@ def __init__(
         config: Sequence[Config],
         stats_rolling_window: int,
         game_buffer_capacity: int,
+        harden_below_visits: Optional[int],
```

Comment:
What would happen if we did:
```suggestion
        harden_below_visits: int = 0,
```

This would avoid needing to explicitly check for None on comparisons like on line 529, but since visits always >1 the comparison would still return True if `harden_below_visits` not explicitly specified. Would this have any negative effects?

(If you do use this change would recommend adding brief explanation to docstring as it may be counterintuitive why it's 0 by default.)